### ISSUE TITLE
[Feature Request] Please support histogram-ish ping in mobile telemetry.

### ISSUE THREAD

**Issue by cnevinc** [mozilla-services/mozilla-pipeline-schemas#190]
As a new product, we want to track new features with new telemetry and maybe remove it once we don't need it. In such a cadence, we'll need to update core ping's schema very often.

Currently, there are two alternatives:
1. Send an event to telemetry_focus_event_parquet when the value has changed: This is not ideal. Cause we want to know the current state of all users, not to count the times they change their state.
2. Put them in telemetry_focus_event_parquet's setting column. This is will be confusing if the state we want to track is not a real setting.

In conclusion, a histogram or some other fields we could use to track things without updating the schema often is what we need.

**Comment by acmiyaguchi** [comment]
This is fulfilled with the introduction of [Glean](https://docs.telemetry.mozilla.org/concepts/glean/glean.html) on various mobile products. This has been used on both Android products and iOS products, and introduces various histogram measurements that are automatically updated in the table schemas based on a metrics yaml file in the source of the instrumented code.
